src/components/charts/bar/BarItemLabel.js

```diff
@@ -54,7 +54,7 @@ export default class BarItemLabel extends Component {
             <g transform={`translate(${x},${y})`} className="nivo_bar_legend" style={labelStyle}>
                 {line}
                 <text x={textX} textAnchor={textAnchor} dy="0.5em" style={{ fill: textColor }}>
-                    {data.value}
+                    {textFormat(data.value)}
```

Comment:
should be `getLabel(data)` as the `getLabelGenerator()` function handles both accessing & formatting the value